clients/js/workshop/session3-langcache/solution/client.ts

```
@@ -0,0 +1,227 @@
+import { LangCache } from "@redis-ai/langcache";
+
+import {
+    AISHE_API_URL,
+    REQUEST_TIMEOUT_MS,
+    APIClientError,
+    aisheAPIRequest,
+    type HealthResponse,
+    type AnswerResponse,
+    LANGCACHE_CLOSE_SIMILARITY_THRESHOLD,
+} from "aishe-client";
+
+// TODO: replace with your own API KEY, CACHE ID and URL
+const LANGCACHE_API_KEY: string = "YOUR_API_KEY";
+const LANGCACHE_CACHE_ID: string = "YOUR_CACHE_ID";
+const LANGCACHE_SERVER_URL: string = "YOUR_SERVER_URL";
```

Why not make these env vars and put them in the config, maybe throw an error if they're not found?